WashU-IT-RIS/coldfront-wustl-fork
# Feature/itdev 34908

https://washu.atlassian.net/browse/ITDEV-34908

To test:
If testing locally, be sure to run `coldfront migrate` from your coldfront-dev container, as there is a migration out now.

Try to add a group to any allocation in  RW or RO, then confirm that you can remove the group.  For additional verification, you can check Active Directory directly via your preferred method to confirm that the group has been added
